zebra/subdir.am

```diff
@@ -242,6 +243,27 @@ zebra_dplane_fpm_nl_la_LDFLAGS = $(MODULE_LDFLAGS)
 zebra_dplane_fpm_nl_la_LIBADD  =
 endif
 
+# Dataplane using protobuf plugin
```

Comment:
shouldn't this be conditional - do we need to build this all the time?

Reply:
Thanks for suggestions. I'll change commits and build options.

Reply:
Done.